fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/runtime/definitions/v14/typeMapping/SiTypeMapping.kt

```diff
@@ -171,13 +173,37 @@ object SiByteArrayMapping : SiTypeMapping {
     }
 }
 
+class ReplaceTypesSiTypeMapping(
+    vararg cases: Pair<String, Type<*>>
+) : SiTypeMapping {
+
+    companion object {
+        fun default(): SiTypeMapping {
+            return ReplaceTypesSiTypeMapping(
+                "sp_runtime.generic.era.Era" to EraType
```

Comment:
We no longer hard-code EraType in SignedExtensions so we need to replace it at type parsing stage to use our own EraType instead of metadata v14 version